tests/errors/SettingsError/filters/f01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:--filter;asdf=2016-02-01;
```

Comment:
Does the library prepend a hyphen automatically? If so, that seems inflexible. What if an option needs to be added which doesn't start with a hyphen?